src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/BasePostsCreator.java

```diff
@@ -92,4 +91,27 @@ public PostsCreator clickSubmitButton() {
     getSubmitButton().click();
     return this;
   }
+
+  @Override
+  public PostEntity.Data addPostWithRandomData() {
```

Comment:
Why Random? if it has to unique - use timestamp or so, if it can be anything, then use fixed values.